boxcli/args.go

```diff
@@ -3,13 +3,19 @@
 
 package boxcli
 
+import "path/filepath"
+
 // Functions that help parse arguments
 
 // If args empty, defaults to the current directory
 // Otherwise grabs the path from the first argument
 func pathArg(args []string) string {
 	if len(args) > 0 {
-		return args[0]
+		p, err := filepath.Abs(args[0])
+		if err != nil {
+			panic(err) // What even triggers this?
```

Comment:
It can fail if the user doesn't have permissions to read part of the path needed to make it absolute.